action.yml

```diff
@@ -65,6 +65,10 @@ inputs:
   headed:
     description: 'Whether or not to use headed mode'
     required: false
+  publish-summary:
+    description: 'Whether or not to publish job summary'
+    required: false
+    default: false
```

Comment:
Maybe I misunderstood the logic, but it seems like setting `publish-summary:` / `default: false` will change the default behaviour for the action.

If the default behaviour is changed, then that would make it a breaking change. Please check out this point. It might make sense to change the default, but in that case I would separate this into two changes:
1. add functionality and keep default
2. later when a new major version is planned, then change the default and document it in a migration guide

Reply:
imo, the default should be true in order to avoid any breaking changes

Reply:
ah I think I understood it the other way around 😅 . If that is the case, sounds like defaulting it to `true` would be the right behavior to avoid a breaking change

Reply:
I've made some suggestion on the PR (you can find them below). Feel free reject or accept them :)